forch/faucet_state_collector.py

```diff
@@ -1025,6 +1025,13 @@ def process_port_state(self, timestamp, name, port, state):
             port_table[PORT_STATE_TS] = datetime.fromtimestamp(timestamp).isoformat()
             port_table[PORT_STATE_COUNT] = port_table.setdefault(PORT_STATE_COUNT, 0) + 1
 
+            if not state:
+                port_attr = self._get_port_attributes(name, port)
+                if port_attr and port_attr['type'] == 'access':
+                    if self._placement_callback:
+                        devices_placement = DevicePlacement(switch=name, port=port, connected=False)
```

Comment:
should this be devices_placement?  The plural/possessive seems weird.

Reply:
Renamed